### PR TITLE
Added Rspec defaults

### PR DESCRIPTION
**Pull request by Mrjaco12**
Why:

We for easy test install

This change addresses the need by:

Running the rspec install and including defaults.

### CODE REVIEW

**Comment by jasontwong** [comment]
:+1: Though @samcdavid probably wants to add [Fuubar](https://github.com/thekompanee/fuubar)

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# Prevent database truncation if the environment is production
+abort('The Rails environment is running in production mode!') if Rails.env.production?
+require 'spec_helper'
+require 'rspec/rails'
+require 'shoulda/matchers'
+
+ActiveRecord::Migration.maintain_test_schema!
+
+RSpec.configure do |config|
+  config.fixture_path = "#{::Rails.root}/spec/fixtures"
```
Comment (samcdavid) [inline spec/rails_helper.rb]: I thought we were using `spec/factories` instead of `spec/fixtures` or am I confused.

**Comment by samcdavid** [comment]
Eh, I like fuubar but I don't think it should be a requirement for raygun.

File: spec/rails_helper.rb (hunk 1 above)
Comment (bbrock25) [inline spec/rails_helper.rb]: i prefer factories :smile:

Comment (Mrjaco12) [inline spec/rails_helper.rb]: Actually we don't use fixtures per se the way rspec expects them. We're covered as long as we have:
`config.include FactoryGirl::Syntax::Methods`

That said, I guess we should remove that line as it's unnecessary. But I haven't tested that so don't know if it'll break anything.

**Comment by bbrock25** [comment]
@Mrjaco12 [here](https://robots.thoughtbot.com/shoulda-matchers-3-0) is the article I mentioned earlier. It has to do with the Shoulda matchers setup, not Rspec. Should we amend this PR to include these defaults (assuming that we are using shoulda)?

**Comment by squaresurf** [comment]
We have houndci!!! I'm so happy :sob:

**Comment by bbrock25** [comment]
can we update the config here to allow single quoted strings? I'm really starting to like them.

**Comment by Mrjaco12** [comment]
@bbrock25 take a look [here](https://github.com/spartansystems/raygun-rails-template/pull/4). Lets get the linting PR in before this one so that we can base this off our default configs.

**Comment by jasontwong** [comment]
@bbrock25 I usually use single quote's cause I'm too lazy to hold the shift for `"`
I'm fine with whatever the linters say as long as I can autocorrect them like `rubocop -a`

**Comment by Mrjaco12** [comment]
So hound will be updated to allow single quotes now. @bbrock25

**Comment by bbrock25** [comment]
yay

**Comment by squaresurf** [comment]
Whatever we decide on quote style we should require it in the linter so that, like @jasontwong said, we can easily have it fixed by rubocop. Not to mention keep consistent style.

**Comment by bbrock25** [comment]
:+1:

**Pull request merged**